src/VisualStudio/Core/Def/Implementation/AbstractVsTextViewFilter.cs

```diff
@@ -29,7 +29,7 @@ internal abstract class AbstractVsTextViewFilter : AbstractOleCommandTarget
         }
 
         // Internal for testing purposes
-        internal static int GetPairExtentsWorker(ITextView textView, Workspace workspace, IBraceMatchingService braceMatcher, int iLine, int iIndex, TextSpan[] pSpan, CancellationToken cancellationToken)
+        internal static int GetPairExtentsWorker(ITextView textView, Workspace workspace, IBraceMatchingService braceMatcher, int iLine, int iIndex, TextSpan[] pSpan, uint? commandId, CancellationToken cancellationToken)
```

Comment:
When does commandId not have a value?  Also, it doesn't look like this method needs the command id, couldn't you make it into a bool (or bool?) with a name that describes what it controls instead of pushing the command into here?